scrapers/RealityKings.yml

```diff
@@ -66,19 +66,29 @@ xPathScrapers:
             with: https://www.realitykings.com
 
   sceneScraper:
+    common:
+      $section: //img[@alt=""][contains(@src,"poster")]/../../.. #img>div>div>section
     scene:
-      Title: //h2[contains(@class, "hzrLcs")]/text()
+      Title: $section//h1/text()|$section//h2/text()
       Date:
-        selector: //div[contains(@class, "jtRRdJ")]/text()
+        selector: //div[text()="Release Date:"]/../text()
         parseDate: January 2, 2006
-      Details: //p[contains(@class, "kLbWwG")]/text()
+      Details: //div[text()="Description:"]/../div[not(@class)]/text()
```

Comment:
```yaml
Details: //section/h2[text()="Description"]/following-sibling::div/p/text()

I was trying this out for the details.

Reply:
this `//section/h2[text()="Description"]/following-sibling::div/p/text()` doesn't seem to work with most of the sites i tested eg  sneakysex, seancody, sweetheartvideo, teenslovehugecocks